packages/async/src/resolver.ts

```diff
@@ -93,14 +93,18 @@ function tryRequire(id: string) {
   ) {
     try {
       return normalize(__webpack_require__(id));
-    } catch {
-      // Just ignore failures
+    } catch (error) {
+      // Log errors
+      // eslint-disable-next-line no-console
+      console.log(`webpack require error: ${error}`);
```

Comment:
I'm open to other suggestions for logging this. I thought about introducing something like `pinojs` but it just seems like overkill.

Reply:
This definitely improves local debugging experience for known SSR problems.  I would like to explore what automated discovery of niche SSR errors looks like, though.  In an ideal world, these errors would be exposed to consumers to let them decide on how/where to log. Probably via `react-server#extract` .

That's likely way more work than it sounds 😄  I'm opening this can o' worms because:
* One SSR error appears to break _all_ future SSR for a pod
* This would allow a niche admin section to quietly break SSR on individual pods without the obvious, cross-pod metrics dip that we see with major SSR bugs

Reply:
> One SSR error appears to break all future SSR for a pod

This is based on me glancing at metrics during bad SSR deploys. I could definitely be wrong.

Reply:
> In an ideal world, these errors would be exposed to consumers to let them decide on how/where to log. Probably via react-server#extract

I agree with this, if we could surface stuff like this through actual runtime errors somewhere like at the extract call we could get a lot more observability at minimum, and maybe even have some automated testing at maximum